oat-sa/extension-tao-mediamanager
# Hide loading icon on the previewer element loaded

Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/UNO-514
 
1. Select an audio file in Assets
2. Observe its preview.
 
#### Steps to reproduce  (for bugs)
 - Load mp3
 - Check that loading icon is still there, in the preview section
  
#### How to test
 
Load mp3 to the Assets, and check that no loading progress presented in the preview section
